File: reni-frontend/src/main/reni-frontend/src/app/modules/intranet/dto/response/perfil-funcionalidad-perf-listar.response.ts
export class PerfilFuncionalidadPerfListarResponse {
    idFuncionalidad: number;
    idFuncionalidadPadre: number;
    titulo: string;
    referencia: string;
    imagen: string;
    orden: number;
    flgActivo: number;
    listaFuncionalidad: PerfilFuncionalidadPerfListarResponse[];
}

File: reni-backend/src/main/java/pe/gob/mimp/reni/dto/response/PerfilFuncionalidadPerfListarResponse.java
package pe.gob.mimp.reni.dto.response;

import java.io.Serializable;
import java.util.List;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@Getter
@Setter
@ToString
public class PerfilFuncionalidadPerfListarResponse implements Serializable {
	private static final long serialVersionUID = 1L;

	private Long idFuncionalidad;
	private Long idFuncionalidadPadre;
	private String titulo;
	private String referencia;
	private String imagen;
	private Integer orden;
	private Integer flgActivo;
	private List<PerfilFuncionalidadPerfListarResponse> listaFuncionalidad;
}
